x-pack/examples/ui_actions_enhanced_examples/public/plugin.ts

```diff
@@ -4,44 +4,174 @@
+import { mount } from './mount';
+import {
+  UiActionsEnhancedMemoryActionStorage,
+  UiActionsEnhancedDynamicActionManager,
+} from '../../../plugins/ui_actions_enhanced/public';
+import { App2ToDashboardDrilldown } from './drilldowns/app2_to_dashboard_drilldown';
 
 export interface SetupDependencies {
+  dashboard: DashboardSetup;
   data: DataPublicPluginSetup;
+  developerExamples: DeveloperExamplesSetup;
   discover: DiscoverSetup;
   uiActionsEnhanced: AdvancedUiActionsSetup;
 }
 
 export interface StartDependencies {
+  dashboard: DashboardStart;
   data: DataPublicPluginStart;
   discover: DiscoverStart;
   uiActionsEnhanced: AdvancedUiActionsStart;
 }
 
+export interface UiActionsEnhancedExamplesStart {
+  managerWithoutEmbeddable: UiActionsEnhancedDynamicActionManager;
+  managerWithoutEmbeddableSingleButton: UiActionsEnhancedDynamicActionManager;
+  managerWithEmbeddable: UiActionsEnhancedDynamicActionManager;
+}
+
 export class UiActionsEnhancedExamplesPlugin
-  implements Plugin<void, void, SetupDependencies, StartDependencies> {
+  implements Plugin<void, UiActionsEnhancedExamplesStart, SetupDependencies, StartDependencies> {
   public setup(
-    core: CoreSetup<StartDependencies>,
-    { uiActionsEnhanced: uiActions }: SetupDependencies
+    core: CoreSetup<StartDependencies, UiActionsEnhancedExamplesStart>,
+    { uiActionsEnhanced: uiActions, developerExamples }: SetupDependencies
   ) {
     const start = createStartServicesGetter(core.getStartServices);
 
     uiActions.registerDrilldown(new DashboardHelloWorldDrilldown());
     uiActions.registerDrilldown(new DashboardHelloWorldOnlyRangeSelectDrilldown());
     uiActions.registerDrilldown(new DashboardToDiscoverDrilldown({ start }));
+    uiActions.registerDrilldown(new App1HelloWorldDrilldown());
+    uiActions.registerDrilldown(new App1ToDashboardDrilldown({ start }));
+    uiActions.registerDrilldown(new App2ToDashboardDrilldown({ start }));
+
+    uiActions.registerTrigger(sampleApp1ClickTrigger);
+    uiActions.registerTrigger(sampleApp2ClickTrigger);
+
+    uiActions.addTriggerAction(SAMPLE_APP2_CLICK_TRIGGER, {
+      id: 'SINGLE_ELEMENT_EXAMPLE_OPEN_FLYOUT_AT_CREATE',
+      order: 2,
+      getDisplayName: () => 'Add drilldown',
+      getIconType: () => 'plusInCircle',
+      isCompatible: async ({ workpadId, elementId }: SampleApp2ClickContext) =>
+        workpadId === '123' && elementId === '456',
+      execute: async () => {
+        const { core: coreStart, plugins: pluginsStart, self } = start();
+        const handle = coreStart.overlays.openFlyout(
+          toMountPoint(
+            h(pluginsStart.uiActionsEnhanced.FlyoutManageDrilldowns, {
+              onClose: () => handle.close(),
+              viewMode: 'create',
+              dynamicActionManager: self.managerWithoutEmbeddableSingleButton,
+              triggers: [SAMPLE_APP2_CLICK_TRIGGER],
+              placeContext: { sampleCanavsElementClickContext: sampleApp2ClickContext },
```

Comment:
I also can't find where `sampleCanavsElementClickContext` if it is not used, let's remove to simplify examples? 
If it is used, let's completely go away from `canvas` `ml` references and use `dummy` references instead

Reply:
Done.